src/pages/billing/Project.js

```diff
@@ -147,41 +147,39 @@ const BillingAccountSummaryPanel = (() => {
     h(StatusAndCount, { status, count })
   ])
 
-  return Utils.memoWithName(
-    'BillingAccountSummaryPanel',
-    ({ counts: { done, error, updating } }) => div({
-      style: {
-        padding: '0.5rem 2rem 1rem',
-        position: 'absolute',
-        top: topBarHeight,
-        right: '3rem',
-        width: '30rem',
-        backgroundColor: colors.light(0.5),
-        boxShadow: '0 2px 5px 0 rgba(0,0,0,0.25)'
-      }
-    }, [
-      div({ style: { padding: '1rem 0' } }, 'Your billing account is updating...'),
-      div({ style: { display: 'flex', justifyContent: 'flex-start' } }, [
-        maybeAddStatus('updating', updating),
-        maybeAddStatus('done', done),
-        maybeAddStatus('error', error)
-      ]),
-      error > 0 && div({ style: { padding: '1rem 0 0' } }, [
-        'Try again or ',
-        h(Link, { onClick: () => contactUsActive.set(true) }, [
-          'contact us regarding unresolved errors'
-        ]), '.'
-      ])
-    ]))
-})()
+  return div({
+    style: {
+      padding: '0.5rem 2rem 1rem',
+      position: 'absolute',
+      top: topBarHeight,
+      right: '3rem',
+      width: '30rem',
+      backgroundColor: colors.light(0.5),
+      boxShadow: '0 2px 5px 0 rgba(0,0,0,0.25)'
+    }
+  }, [
+    div({ style: { padding: '1rem 0' } }, 'Your billing account is updating...'),
+    div({ style: { display: 'flex', justifyContent: 'flex-start' } }, [
+      maybeAddStatus('updating', updating),
+      maybeAddStatus('done', done),
+      maybeAddStatus('error', error)
+    ]),
+    error > 0 && div({ style: { padding: '1rem 0 0' } }, [
+      'Try again or ',
+      h(Link, { onClick: () => contactUsActive.set(true) }, [
+        'contact us regarding unresolved errors'
+      ]), '.'
+    ])
+  ])
+}
 
 const groupByBillingAccountStatus = (billingProject, workspaces) => {
-  const group = workspace => Utils.cond(
-    [billingProject.billingAccount === workspace.billingAccount, () => 'done'],
-    ['billingAccountErrorMessage' in workspace, () => 'error'],
-    [Utils.DEFAULT, () => 'updating']
-  )
+  const group = workspace => billingProject.billingAccount === workspace.billingAccount ? 'done' :
+    !!workspace.billingAccountErrorMessage ? 'error' :
+      'updating'
```

Comment:
I hear you, but we really try to avoid nested ternaries. I'd prefer to see it as a cond because that's our standard in this codebase, and we use it everywhere else. I want to keep things consistent.

Reply:
General question: why should we be preferring C-style syntax over clojure-style syntax?

I am happy with the argument in this case that it preserves evaluation order, that is compelling to me personally, but I don't necessarily find C that readable and the nice thing about the clojure syntax is it makes it a little bit more clear to me what the possible outcomes are, whereas if we got more nested ternaries in here (two is okay, but I have seen it get much longer than that), I personally start to have trouble making that out. It takes what was a relatively easy task and makes it difficult.

Reply:
Parking this conversation for a UI meeting. Lots of good comments.Reverting to cond.